### ISSUE TITLE
Not clear that Item Templates are not supported in the Visual Studio UI

### ISSUE THREAD

**Issue by james1301** [dotnet/docs#32934]
Hi, I have finally found out that any UI for the Item Templates in Visual Studio is not supported. This is not mentioned in this documentation and I have spent much time trying to get this to work.

The project templates are seen in the New Project view, but Item templates are not in the Item Template section.

This information is according to this: https://github.com/sayedihashimi/template-sample/issues/10.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: d9cb6bd4-1194-91fa-014c-2e72d809e772
* Version Independent ID: c3fddda9-24c1-f20c-a1b6-6aeea61d6e53
* Content: [Create an item template for dotnet new - .NET CLI](https://learn.microsoft.com/en-us/dotnet/core/tutorials/cli-templates-create-item-template)
* Content Source: [docs/core/tutorials/cli-templates-create-item-template.md](https://github.com/dotnet/docs/blob/main/docs/core/tutorials/cli-templates-create-item-template.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @adegeo
* Microsoft Alias: **adegeo**

---
[Associated WorkItem - 52194](https://dev.azure.com/msft-skilling/Content/_workitems/edit/52194)

**Comment by adegeo** [comment]
Hi @james1301 I looked at your referenced issue and some others related to it, does adding `ide.host.json` make the item template visible? I've not tried it but I was wondering if you did. Thanks

**Comment by james1301** [comment]
Hi, I have tried it, but no luck, but that information is a bit old and I am now on a later version of Visual Studio. Either way though it feels the documentation should be highlighting this in some way, as you have seen on that issue it is not recommended and liable to change.

I have reached out on a new issue to try and find out some up to date information, but feels like it is not implemented yet.
